### ISSUE TITLE
Wrong requeue order test

### ISSUE THREAD

**Issue by erfan-mehraban** [arcana261/ubroker#10]
It seems checking requeue order has logical problem.
https://github.com/arcana261/ubroker/blob/cb32497697b58f4bf707abb2d111bc5b12347be0/internal/broker/core_test.go#L248
If we consume after publishing queue is like `(front of queue) 1 2 3 (back of queue)`, requeue in order of `3 2 1` should results in this way:

requeue item | queue
--- | ---
3 | `3`
2 | `2 3`
1 | `1 2 3`

But this function test it in wrong order. (`3 2 1`)

**Comment by arcana261** [comment]
@erfan-mehraban You're right.. we should have prioritized items based on their ID's when we re queued. But I'm afraid it's too late to change that behavior :'(

**Comment by erfan-mehraban** [comment]
@arcana261  Can we correct the test on PR?

**Comment by arcana261** [comment]
Yup do as you please :)
